scripts/crawl_zenodo.py

```diff
@@ -286,13 +291,16 @@ def create_new_dataset(dataset, token, force, username):
     d.no_annex("DATS.json")
     d.no_annex("README.md")
     d.no_annex(".conp-zenodo-crawler.json")
+    d.config.add("datalad.log.timestamp", "true")
+    d.save()
 
     r = d.create_sibling_github(repo_title,
                                 github_login=token,
                                 github_passwd=token)
 
     for bucket in dataset["files"]:
         download_file(bucket, d, dataset_dir)
+    d.save()
```

Comment:
Why this `d.save` and the one on line 295?

Reply:
You are right, let me refactor the code